pylsy/pylsy.py

```diff
@@ -81,31 +82,38 @@ def _print_head(self):
     def _print_value(self):
         for line in range(self.Lines_num):
             for col, length in zip(self.Table, self.AttributesLength):
+                vals = list(col.values())[0]
+                val = vals[line] if len(vals) != 0 and line < len(vals) else ''
                 self.StrTable += "| "
-                value_length = length * 2 - 1
-                value = list(col.values())[0]
-                if len(value) != 0:
-                    if line < len(value):
-                        start = (value_length - len(value[line])) // 2
-                        for space in range(start):
-                            self.StrTable += " "
-                        self.StrTable += value[line]+' '
-                        end = value_length - start - len(value[line])
-                        for space in range(end):
-                            self.StrTable += " "
-                    else:
-                        start = 0
-                        end = value_length - start + 1
-                        for space in range(end):
-                            self.StrTable += " "
-                else:
-                    start = 0
-                    end = value_length - start + 1
-                    for space in range(end):
-                        self.StrTable += " "
+                self.StrTable += self._pad_string(val, length * 2)
             self.StrTable += "|"+'\n'
             self._print_divide()
 
+    def _disp_width(self, pwcs, n=None):
```

Comment:
If I want to make this code short (and perhaps more Pythonic.. hey what is Pythonic?):
```Python
    def _disp_width(self, pwcs, n=None):
        end = len(pwcs) if n is None else n
        idx = slice(0, end)
        return sum([max(0, wcwidth(char)) for char in pwcs[idx]])
```
Not wrapping `idx` inside since it's a func call (internal though)..